LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-event.https.html

```diff
@@ -147,6 +147,28 @@
             'Service Worker should respond with an empty resulting client id for subresource requests');
         });
   }, 'Service Worker responds to fetch event with the correct resulting client id');
+  
+  promise_test(t => {
+      const page_url = 'resources/simple.html?replacesClientId';
+      const expected_found = 'Replaces Client ID Found';
+      const expected_not_found = 'Replaces Client ID Not Found';
+      return with_iframe(page_url)
+        .then(function(frame) {
+            t.add_cleanup(() => { frame.remove(); });
+            assert_equals(
+              frame.contentDocument.body.textContent.substr(0, expected_not_found.length),
+              expected_not_found,
+              'Service Worker should respond with an empty replaces client id for non-subresource requests');
+            return frame.contentWindow.fetch('resources/other.html?replacesClientId');
+          })
+        .then(function(response) { return response.text(); })
+        .then(function(response_text) {
+            assert_equals(
+              response_text.substr(0),
+              expected_not_found,
+              'Service Worker should respond with a replaces client id for subresource requests');
```

Comment:
NOTE: Now that I look at it here, I think "navigation requests" would be more appropriate in the description...